packages/kbn-ui-shared-deps/scripts/build.js

```diff
@@ -38,10 +41,38 @@ run(
     );
 
     /** @param {webpack.Stats} stats */
-    const onCompilationComplete = (stats) => {
+    const onCompilationComplete = async (stats) => {
       const took = Math.round((stats.endTime - stats.startTime) / 1000);
 
       if (!stats.hasErrors() && !stats.hasWarnings()) {
+        if (!flags.dev) {
+          const reporter = CiStatsReporter.fromEnv(log);
+
+          const metrics = [
+            {
+              group: '@kbn/ui-shared-deps asset size',
+              id: 'kbn-ui-shared-deps.js',
+              value: Fs.statSync(Path.resolve(DIST_DIR, 'kbn-ui-shared-deps.js')).size,
+            },
+            {
+              group: '@kbn/ui-shared-deps asset size',
+              id: 'kbn-ui-shared-deps.@elastic.js',
+              value: Fs.statSync(Path.resolve(DIST_DIR, 'kbn-ui-shared-deps.@elastic.js')).size,
+            },
+            {
+              group: '@kbn/ui-shared-deps asset size',
+              id: 'css',
+              value:
+                Fs.statSync(Path.resolve(DIST_DIR, 'kbn-ui-shared-deps.css')).size +
+                Fs.statSync(Path.resolve(DIST_DIR, 'kbn-ui-shared-deps.v7.light.css')).size,
```

Comment:
👍 I think just tracking the current default CSS (v7 light) is the best solution for the long term. I am trying to think of a reason to also track dark vs light and v7 vs v8 but can't think of any logical reason, especially considering it would add noise since they are pretty closely related.

Reply:
Yeah, I also expect that once we switch to v8 we'll remove the v7 theme and this will break, forcing us to move over to the v8 theme. And I think what really matters is the default experience.